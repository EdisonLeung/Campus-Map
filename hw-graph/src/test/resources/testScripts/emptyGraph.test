# Test the creation of an empty graph

# Create an empty graph
CreateGraph emptyGraph

# Print the nodes in the empty graph
ListNodes emptyGraph