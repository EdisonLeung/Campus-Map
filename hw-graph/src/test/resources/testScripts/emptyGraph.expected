# Test the creation of an empty graph

# Create an empty graph
created graph emptyGraph

# Print the nodes in the empty graph
emptyGraph contains: